assets/_sass/custom-viz/boeing/_scrollytelling.scss

```diff
@@ -5,14 +5,14 @@
 
   .svgPart {
     opacity: 0.2;
-    transition: opacity 0.7 linear;
+    transition: opacity 0.7s linear;
```

Comment:
Can we speed this up to .4s? Same with the transitions on `.svgLabel`? .7s is a little bit too long.

Reply:
Yep!